LICENSE.md

```diff
@@ -0,0 +1,22 @@
+MIT License
+
+Copyright (c) 2022 David Hebbeker
```

Comment:
@FiveTeethless, please add your full name here if you want to.

Reply:
done